app/models/rbac.rb

```diff
@@ -302,17 +302,17 @@ def self.minimum_columns_for(klass)
     columns.join(', ')
   end
 
-  def self.get_user_info(userid, miq_group_id = nil)
-    user                      = User.find_by_userid(userid)
-    miq_group                 = MiqGroup.find_by_id(miq_group_id)
+  def self.get_user_info(user, userid, miq_group, miq_group_id)
```

Comment:
@chessbyte I was going to do in multiple PRs.
Thought this change is scary enough to have on it's own.
Especially since it is fixing a bug in the `ids_via_descendants` call.

1. Add ability to pass in `:user` (and keep legacy `:user_id`) <-- THIS PR.
2. Change all the callers to use `:user` where it is easy.
3. Additional changes to using `:user` where this proves to be difficult.
3. Remove `:user_id`.